README.md

```diff
@@ -586,9 +615,20 @@ To update kdb VS Code settings, search for **kdb** from _Preferences_ > _Setting
 | **Hide subscription to newsletter after first install**        | yes/no; default no                                                  |
 | **Insights Enterprise Connections for Explorer**               | [edit JSON settings](#insights-enterprise-connections-for-explorer) |
 | **Linting**                                                    | Enable linting for q and quke files                                 |
+| **Refactoring**                                                | Choose refactoring scope                                            |
 | **QHOME directory for q runtime**                              | Display location path of q installation                             |
 | **Servers**                                                    | [edit JSON settings](#servers)                                      |
 
+### Refactoring
+
+Refactorings such as renaming are by default applied to all files in the workspace. You can preview the changes before applying them and select the files to apply the refactoring by pressing **ctrl** or **command** key before the action.
+
+![Preview](https://github.com/KxSystems/kx-vscode/blob/main/img/preview.png?raw=true)
+
+If you only need to apply the refactorings for the currently opened files, refactoring option **Window** can be selected instead of **Workspace**:
```

Comment:
```suggestion
If you only need to apply the refactorings to the currently opened files, you can select **Window** instead of **Workspace** for the refactoring option:
```